src/docs/product/projects/index.mdx

```diff
@@ -6,25 +6,37 @@ description: "View all the projects associated with teams that you're a member o
 
 A _Project_ represents your service or application in Sentry. For example, you might have separate projects for your API server and frontend client.
 
-You can [create a project](/product/sentry-basics/integrate-frontend/create-new-project/) for a service and select the particular language or framework when you first start using Sentry. You can add more projects on an as-needed basis, for example, if you want to manage access permissions and/or control privacy and data settings. Setting up multiple projects to reflect your application landscape with finer granularity will go a long way in helping with visibility and triage. Learn more about best practices for setting up projects in our [account setup documentation](/product/accounts/getting-started/#4-create-projects).
+When you first [create a project](/product/sentry-basics/integrate-frontend/create-new-project/), you also assign a team to it. All members of the assigned team have access to the project. Additional teams can be added in **[Project] > Settings > Project Teams**.
 
-When displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, you are looking at information that is immediately relevant to your work.
+You can create more projects to reflect your application landscape with finer granularity (for example, a separate project for each repo, mapping components and micro-services to their owning teams). For most pages in Sentry, including **Issues** and **Discover**, the top-level filter bar defaults to showing information for your projects, so that you only see what's relevant to you and your team(s). How you set up your projects can go a long way in reducing noise and improving productivity. Learn more about best practices for setting up projects in our [account setup documentation](/product/accounts/getting-started/#4-create-projects).
```

Comment:
```suggestion
You can create more projects to reflect your application landscape with finer granularity (for example, a separate project for each repo, mapping components and micro-services to the teams that own them). For most pages in Sentry, including **Issues** and **Discover**, the top-level filter bar defaults to showing information for your projects, so that you only see what's relevant to you and your team(s). How you set up your projects can go a long way in reducing noise and improving productivity. Learn more about best practices for setting up projects in our [account setup documentation](/product/accounts/getting-started/#4-create-projects).
```